Assets/Plugins/Source/Editor/ConfigEditors/ConfigEditor.cs

```diff
@@ -214,11 +216,16 @@ private static float GetMaximumLabelWidth(IEnumerable<(FieldInfo, ConfigFieldAtt
         /// </exception>
         protected void RenderConfigFields()
         {
-            var fieldGroups = GetFieldsByGroup();
-            foreach (var fieldGroup in fieldGroups)
+            foreach (var fieldGroup in GetFieldsByGroup())
             {
                 float labelWidth = GetMaximumLabelWidth(fieldGroup);
 
+                // If there is a label for the field group, then display it.
+                if (_groupLabels?.Length > fieldGroup.Key)
```

Comment:
silly nit that you don't really need to address; if fieldGroup.Key is negative, this'd break. Sometimes people put "-1" in Unity ordering things to automatically sort them.

Reply:
Excellent point! This actually could have caused a problem, because the default value if the group number is not set is actually `-1`, so I've updated the implementation to include that in the conditional.